jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/TargetImageParameters.java

```diff
@@ -33,28 +36,27 @@
 public class TargetImageParameters {
 
   private final AuthParameters auth;
+  private final Project project;
 
-  @Nullable private String image;
+  private Property<String> image;
   private Set<String> tags = Collections.emptySet();
   @Nullable private String credHelper;
 
   @Inject
-  public TargetImageParameters(ObjectFactory objectFactory) {
+  public TargetImageParameters(ObjectFactory objectFactory, Project project) {
```

Comment:
No need to accept `objectFactory`, as you can get it by `project.getObjects()`, which is what you are already doing in the new code.